src/Symfony/Component/FeatureFlag/DataCollector/FeatureCheckerDataCollector.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\FeatureFlag\DataCollector;
```

Comment:
I comment such thing sometime but on other components such classes are in a Debug namespace 
Perhaps need to check what to do here :)

Reply:
We should do it like in the other components

Reply:
I was not sure of the structure to use, so I looked at the Serializer and Workflow components :
* Symfony\Component\Serializer
  * Debug
    * TraceableEncoder
    * TraceableNormalizer
    * TraceableSerializer
  * DataCollector
    * SerializerDataCollector
* Symfony\Component\Workflow
  * Debug
    * TraceableWorkflow
  * DataCollector
    * WorkflowDataCollector

Reply:
Oki got it perhaps i mislead the datacollector and other debug traceable
But perhaps it could be right to also place datacollector inside debug as it is related to debug
But lets see core maintainers ideas